client/src/pages/Reusable.js

```diff
@@ -29,7 +29,7 @@ const Reusable = (props) => {
           <Kidsroom />
         </Route>
         <Route path="/">
-          <Div>
+          <Div activeClassName="active">
             <Link to={`${url}/Kitchen`}>Kitchen</Link>
             <Link to={`${url}/Bathroom`}>Bathroom</Link>
             <Link to={`${url}/Livingroom`}>Livingroom</Link>
```

Comment:
```suggestion
            <Link to={`${url}/kitchen`}>Kitchen</Link>
            <Link to={`${url}/bathroom`}>Bathroom</Link>
            <Link to={`${url}/livingroom`}>Livingroom</Link>
```

Routes are usually defined lower case (beware to also change this in your Routes!